website/content/en/docs/_index.md

```diff
@@ -1,14 +1,20 @@
 
 ---
-title: "Documentation"
-linkTitle: "Docs"
+title: "v2 Documentation"
+linkTitle: "v2 Docs"
 weight: 20
 menu:
   main:
     weight: 20
     pre: <i class='fas fa-book'></i>
 ---
 
+New in Karpenter v2:
```

Comment:
I know this is a joke, but I'd keep this out of PRs so customers don't get confused :)